app/assets/stylesheets/application.css.scss

```diff
@@ -230,14 +232,23 @@ body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
 .thought-headline{
   background-repeat: no-repeat;
   background-size: cover;
-  height: 150px;
-  width: 100vw;
-  position: relative;
-  left: -150px;
-  box-shadow: inset 0 -12px 100px 10px rgba(0, 0, 0, 0.4);
-
-    a {
+  color: $active-dark;
+  border-bottom: 1px solid $active-dark;
+  /* height: 150px; */
+  width: 80%;
+/*  position: relative;*/
+  /* left: -150px; */
+  /*box-shadow: inset 0 66px 100px 10px rgba(0, 0, 0, 0.6);*/
+  .right-headline{
+    display: inline-block;
+    .title{
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space